test/functionalTest/cases/0000_large_requests/notification_different_sizes.test

```diff
@@ -241,10 +241,12 @@ Date: REGEX(.*)
 1
 ** Notification sizes:
 Sending message 1 to HTTP server: sending message of REGEX((19490|19491|19492|19493|19494|19495|19496)) bytes to HTTP server
-Sending message 2 to HTTP server: sending message of REGEX((21491|21492|21493|21494|21495|21496|21497)) bytes to HTTP server
+Sending message 2 to HTTP server: sending message of REGEX((21490|21491|21492|21493|21494|21495|21496|21497)) bytes to HTTP server
```

Comment:
Version numbers has passed from 1.15.0 to 2.0.0. That is, one less char. Maybe the need for this fix is due to that.

NTC (informative)